bears/general/TextLintBear.py

```diff
@@ -0,0 +1,194 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.settings.Setting import typed_list
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+from dependency_management.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='textlint',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+):(?P<column>\d+)[\s*|✓]*(?P<severity>'
```

Comment:
I know python supports unicode, though I wouldn't stress that support on different platforms. Is it possible to write the checkmark character as some kind of unicode-code? Like `\x0030` or so?